RetailCoder.VBE/UI/CodeInspections/InspectionResultsViewModel.cs

```diff
@@ -283,14 +285,24 @@ private void ExecuteCopyResultsCommand(object parameter)
             {
                 return;
             }
+            var aResults = _results.Select(result => result.ToArray()).ToArray();
 
-            var results = string.Join("\n", _results.Select(result => result.ToString() + Environment.NewLine).ToArray());
             var resource = _results.Count == 1
                 ? RubberduckUI.CodeInspections_NumberOfIssuesFound_Singular
                 : RubberduckUI.CodeInspections_NumberOfIssuesFound_Plural;
-            var text = string.Format(resource, DateTime.Now, _results.Count) + Environment.NewLine + results;
 
-            _clipboard.Write(text);
+            var title = string.Format(resource, DateTime.Now.ToString(CultureInfo.InstalledUICulture), _results.Count);
+
+            var textResults = title + Environment.NewLine + string.Join("", _results.Select(result => result.ToString() + Environment.NewLine).ToArray());
+            var csvResults = ExportFormatter.Csv(aResults, title);
+            var htmlResults = ExportFormatter.HtmlClipboardFragment(aResults, title);
```

Comment:
Ah, there's the coupling. If `InspectionResultsViewModel` took an `IExportFormatter` constructor parameter, its constructor would document the fact that the object is able to format/export its results.